testing/miniflux/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Frédéric Guillot <fred@miniflux.net>
+# Maintainer: Frédéric Guillot <fred@miniflux.net>
+pkgname=miniflux
+pkgver=2.0.16
+pkgrel=0
+pkgdesc="Minimalist and opionated feed reader"
+url="https://miniflux.app/"
+arch="all"
+license="Apache-2.0"
+depends=""
+makedepends="go"
+install="$pkgname.pre-install"
+subpackages="$pkgname-openrc $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/miniflux/miniflux/archive/${pkgver}.tar.gz
+	miniflux.initd
+	miniflux.confd
+	miniflux.conf
+	miniflux.logrotate
+	"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make miniflux VERSION=$pkgver
+}
+
+check() {
+	cd "$builddir"
+	./miniflux -version > /dev/null
```

Comment:
please use the upstream test suite or disable check with a comment explaining.

Reply:
Fixed